docs/data-sources/object_storage_account.md

```
@@ -0,0 +1,82 @@
+```terraform
+data "katapult_object_storage_account" "main" {
+  region = "uk-lon-1"
+}
+
+resource "katapult_object_storage_bucket" "assets" {
+  name                      = "my-org-assets"
+  object_storage_account_id = data.katapult_object_storage_account.main.id
+}
+```
+
+## Example Usage
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)